Feat: Introduce Divider component #DS-1391 (#1552)
lmc-eu/spirit-design-system: packages/web-react/src/components/Divider/README.md

# Divider

> The `Divider` component is used to separate content in a layout.

Reviewer: What about

```suggestion
The `Divider` component is used as horizontal separator of the content.
```
?
